scripts/plugins/openvswitch-config-update

```diff
@@ -91,6 +92,16 @@ def update(session, args):
         new_controller = True
         ret_str += 'Successfully set controller to %s.  ' % controller
 
+    # Open port required by the SDN Controller
+    if controller_local_ports:
+        for local_port in controller_local_ports:
+            splitted = local_port.split(':')
+            protocol = splitted[0]
+            port = splitted[1]
+            cmd = [iptablectl, 'open', port, protocol]
+            output = subprocess.Popen(cmd, stdout=subprocess.PIPE).communicate()
+
```

Comment:
I'm not sure I put it in the right place. Just tried it and the port didn't open.
I thought the `update` method of this script would be called when the SDN Controller fields was updated.